### DISCUSSION TITLE
import bookmarks to Group Speed Dial

### DISCUSSION THREAD

**Discussion by Alexnoj** [fastaddons/GroupSpeedDial#291] [Q&A] [answered]
How can I import all my bookmarks in Vivaldi to Group Speed Dial?

**Comment by Juraj-Masiar** [comment, marked as answer]
Hello,
When you first install my speed dial, it will allow you to import your bookmarks as dials.
Alternatively, if you can export your bookmarks as HTML file, I think you will be able to import it too, from the Options page / Import backup tab / "Import dials from Chrome bookmarks ".html" file"
